delegates.txt

```diff
@@ -373,6 +373,7 @@ Niko Matsakis (NM)
 Nikolaus Papaspyrou (NPU)
 Noah Tye (NTE)
 Norbert Lindenberg (NL)
+Olivier Flückiger (OF)
```

Comment:
New delegates should have a 3-letter acronym and it's recommended to take the first letter of the first name and the first and last letters of the last name, if applicable and available.

```suggestion
Olivier Flückiger (OFR)
```